doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -301,7 +301,9 @@ Debug libraries
 Modem libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`pdn_readme` library:
+
+  * Updated the library to allow PDP_TYPE only configuration in the :c:func:`pdn_ctx_configure` function.
```

Comment:
```suggestion
  * Updated the library to allow a ``PDP_TYPE``-only configuration in the :c:func:`pdn_ctx_configure` function.
```

Reply:
OK, I'll use "PDP_type" to align with AT manual in exact way